README.md

```diff
@@ -8,9 +8,9 @@ Create an ECR repository in your AWS account. In this example, we'll assume it's
 
 1. Clone the repo and set the following environment variables:
     ```shell script
-    export ACCOUNT_ID=<YOUR_ACCOUNT_ID>
     export REGION=us-west-2
     export REPOSITORY_NAME=braket-base-jobs
+    export ACCOUNT_ID=<YOUR_ACCOUNT_ID>
```

Comment:
I made this change so it's easier to copy and paste and run

Reply:
For my info, how does it make it easier?

Reply:
If you copy and paste from the README into your terminal, it will automatically run the first two lines (because of the newlines) and paste the last into the terminal ready to go. This way, if you're using the default region/repo name, you can just change the last command. Otherwise, you have to copy and paste the commands in batches or find some other work around. Even if it was all on one line, it's easier to replace the suffix of a string rather than scrolling earlier. It's a small change, but a nice QOL improvement

Reply:
Good call! I didn't think of that.